### PR TITLE
docs(Packages): Update Community Maintained Packages

### PR DESCRIPTION
**Pull request by StevenMcElligott**
Renamed some options, added another one since TrueCharts doesn't do SCALE anymore and only HELM Charts

### CODE REVIEW

File: content/en/docs/Installation/packages.md [hunk 1]
```diff
@@ -28,10 +28,11 @@ More packages available, with links to download/install instructions:
 |-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Cloudron                | https://www.cloudron.io/store/org.navidrome.cloudronapp.html                                                                                                                  |
 | Fedora                  | https://copr.fedorainfracloud.org/coprs/lchh/navidrome/                                                                                                                       |    
-| Kubernets Chart         | https://artifacthub.io/packages/helm/emmas-charts/navidrome                                                                                                                   | 
+| Helm Chart              | https://artifacthub.io/packages/helm/emmas-charts/navidrome                                                                                                                   |
```
Comment (deluan) [inline content/en/docs/Installation/packages.md:31]: This one has a really old version, 0.50.0. Is it maintained at all?

File: content/en/docs/Installation/packages.md [hunk 2]
```diff
@@ -28,10 +28,11 @@ More packages available, with links to download/install instructions:
...
+| TrueCharts Helm Chart   | https://truecharts.org/charts/stable/navidrome/                                                                                                                               | 
 | OpenMediaVault          | [Instructions using docker-compose](https://forum.openmediavault.org/index.php?thread/36635-how-to-install-navidrome-using-docker-compose-an-airsonic-booksonic-alternative/) |
 | QNAP                    | https://www.myqnap.org/product/navidrome/                                                                                                                                     |
-| TrueNAS SCALE           | https://truecharts.org/charts/stable/navidrome/                                                                                                                               |
+| TrueNAS SCALE           | https://www.truenas.com/docs/truenasapps/communityapps/navidrome/                                                                                                             |
```
Comment (deluan) [inline content/en/docs/Installation/packages.md:35]: This seems to be just a placeholder, there's no information on how to install Navidrome. Don't think it is worth to keep it.

**Review by deluan — COMMENTED** [review]
Thanks for the PR, see my comments below.

File: content/en/docs/Installation/packages.md (hunk 1 above)
Comment (StevenMcElligott) [reply content/en/docs/Installation/packages.md:31]: I honestly don't think so, I've used TrueCharts and the Helm Chart so I know it works, is maintained

File: content/en/docs/Installation/packages.md (hunk 2 above)
Comment (StevenMcElligott) [reply content/en/docs/Installation/packages.md:35]: They don't have much info on the Website for installation but I think it's a massive plus for you guys to list it. It's featured among their catalog apps and is well maintained by @stavros-k 

image

And their installer is pretty straightforward

image

File: content/en/docs/Installation/packages.md (hunk 1 above)
Comment (StevenMcElligott) [reply content/en/docs/Installation/packages.md]: Removed @deluan , let me know if anything else is needed!

**Comment by deluan** [comment]
Thanks!

**Pull request merged**